presto-function-namespace-managers/src/main/java/com/facebook/presto/functionNamespace/execution/SqlFunctionExecutors.java

```diff
@@ -41,7 +42,7 @@
     public SqlFunctionExecutors(Map<Language, FunctionImplementationType> supportedLanguages, ThriftSqlFunctionExecutor thriftSqlFunctionExecutor)
     {
         this.supportedLanguages = requireNonNull(supportedLanguages, "supportedLanguages is null");
-        this.thriftSqlFunctionExecutor = requireNonNull(thriftSqlFunctionExecutor, "thriftSqlFunctionExecutor is null");
+        this.thriftSqlFunctionExecutor = thriftSqlFunctionExecutor;
```

Comment:
Is it possible to inject `Optional`? If it is not possible to inject `Optional` could we inject it via a setter and make the field `Optional` at least?